packages/genesys-spark-components/src/components/beta/gux-side-sheet/example.html

```diff
@@ -0,0 +1,73 @@
+<div class="side-sheet-content">
+  <h1>Side Sheet</h1>
+</div>
+
+<gux-side-sheet-beta size="medium">
+  <gux-icon slot="icon" icon-name="fa/diamond-regular" size="medium"></gux-icon>
+  <gux-side-sheet-heading-beta slot="heading" level="3">
+    Side sheet title
+  </gux-side-sheet-heading-beta>
```

Comment:
I dont think this component needs the `beta` tag tbh since its just styling the heading ? Also I am sort of confused by this component. I cannot find anywhere in the designs where it specifies the use of heading tags.

Could you not just have a component that has a default slot and the user can slot whatever they like eg a heading tag and you style whatever element is passed with the same styling like,


```
::slotted(*) {
  padding: 0;
  margin: 0;
  font-family: var(--gse-ui-dynamicSideSheet-heading-fontFamily);
  font-size: var(--gse-ui-dynamicSideSheet-heading-fontSize);
  font-weight: var(--gse-ui-dynamicSideSheet-heading-fontWeight);
  line-height: var(--gse-ui-dynamicSideSheet-heading-lineHeight);
  color: var(--gse-ui-dynamicSideSheet-headerColor);
}
```

so regardless of what the consumer slots it will always get the same styling so the api might look like,
`<gux-side-sheet-heading slot="heading"><h1>This is a heading</h1> </gux-side-sheet-heading>`

There may be an issue if a user was to slot just a text node as `::slotted(*)` cannot target that but you could just create a `light.scss` file and do it there like,

```
gux-side-sheet-beta {

gux-side-sheet-heading {
   ... same styles as from the component

}
}
```

I may have missed something from the design so let me know.

Reply:
@gavin-everett-genesys
**"I cannot find anywhere in the designs where it specifies the use of heading tags."**
UX just specifies a heading but we should allow for different heading levels so people can create the correct document outline for accessibility, so we shouldn't hardcode a heading level like in the modal.

**"default slot and the user can slot whatever they like eg a heading tag"**
Yes, that would be ideal & I was originally slotting in  h1, h2, h3...tags but there are global styles on the h* tags that are taking precedence. I'd like to avoid light dom styles here if at all possible.

**"issue if a user was to slot just a text node "**
There will have to be some tag present if even just to be able to apply the slot attribute so I don't think you can just insert a text node without some sort of wrapper.

Ping me if you have a better idea & we can tinker with it.